bindings/python/src/_pyghex/py_dtype_to_cpp_name.cpp

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Not really happy about the name and a new file. I'm open for suggestions, but also ok with this.